### PR TITLE
Implements player history methods

### PR DESCRIPTION
**Pull request by JJendryka**
Implements player history methods: `set_history` and `has history`

### CODE REVIEW

**Comment by JJendryka** [comment]
[Implement history related methods in Player class](https://trello.com/c/lnBvhA3b/30-implement-history-related-methods-in-player-class)

**Comment by Hazurl** [comment]
Maybe it should manipulate dictionary like for traits and items ? 🤔

**Comment by Jackojc** [comment]
+1 on that, Hazurl

File: src/player.py [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Player(object):
     """Describe the player, his inventory, traits, position and history"""
 
     def __init__(self, state):
-        pass
+        self.history = {}
```
Comment (Jackojc) [inline src/player.py]: Add:
```
def __init__(self, state = None):
    self.state = state
```

**Pull request merged**